src/native-common/WebView.tsx

```diff
@@ -41,6 +41,7 @@ export class WebView extends RX.WebView<{}> {
                 scalesPageToFit={ this.props.scalesPageToFit }
                 onError={ this.props.onError }
                 onLoadStart={ this.props.onLoadStart }
+                onMessage={this.props.onMessage}
```

Comment:
This prop is not defined on the WebViewProps object within Types.ts. It's also not clear how this same prop would be supported in the web implementation. In general, all props that are supported in ReactXP should be implemented in all platforms. All props should also be documented in the components/docs directory (in this case, in the components/docs/webview.md file).